laws/views.py

```diff
@@ -873,6 +908,30 @@ def knesset_proposal_auto_complete(request):
 
     return HttpResponse(json.dumps(result), mimetype='application/json')
 
+@require_http_methods(["GET"])
+def committee_meeting_auto_complete(request):
+    if not request.GET.get('query'):
+        raise Http404
+
+    q = request.GET['query']
+
+    options = CommitteeMeeting.objects.filter(
+        Q(date_string__icontains=q) | Q(topics__icontains=q)
+    )[:30]
```

Comment:
Why [:30] seems granted to miss some, and we have no option in UI to fix. unless there is a "more committees" options integrated with scroll down behavior